assets/js/components/Loadpoints/Loadpoint.vue

```diff
@@ -11,14 +11,24 @@
 					<div class="text-truncate">
 						{{ loadpointTitle }}
 					</div>
+					<PrioButton
+						v-if="priority || (effectivePriority && effectivePriority !== priority)"
+						:prio="
+							effectivePriority && effectivePriority !== priority
+								? effectivePriority
+								: priority
+						"
+						:editable="effectivePriority === priority"
```

Comment:
zu viel (redundante) Logik im Template. Ich würd `priority` und `effectivePriority` als Props in die Komponente geben. Die muss dann selbst die richtigen Werte ableiten.

Ob die Komponente angezeigt wird gehört weiterhin hier in den Loadpoint. Hier würde ich die Regel machen, dass wir das nicht pro Loadpoint entscheiden sondern bereit auf der Ebene darüber (Loadpoints). Die gibt dann ein `showPriority` an alle Loadpoint Komponenten. Logik wäre: Sobald ein Loadpoint einen nicht `0` Priorität hat wird das Element an allen Loadpoints angezeigt.